test/test_solvers.jl

```diff
@@ -11,1175 +11,139 @@ function test_solvers(FC)
-    @test issolved(solver)
+  for (method, solver) in solvers
+    @testset "$(method)" begin
+      for i = 1 : 3
+        A  = i * A
+        Au = i * Au
+        Ao = i * Ao
+        b  = 5 * b
+        c  = 3 * c
+
+        if method ∈ (:cg, :cr, :symmlq, :minres, :minres_qlp, :cg_lanczos, :diom, :fom,
+                     :dqgmres, :gmres, :fgmres, :cgs, :bicgstab, :bilq, :qmr, :cg_lanczos_shift)
+          method == :cg_lanczos_shift ? solve!(solver, A, b, shifts) : solve!(solver, A, b)
+          niter = niterations(solver)
+          @test Aprod(solver) == (method ∈ (:cgs, :bicgstab) ? 2 * niter : niter)
+          @test Atprod(solver) == (method ∈ (:bilq, :qmr) ? niter : 0)
+          @test solution(solver) === solver.x
+          @test nsolution(solver) == 1
+        end
+
+        if method ∈ (:cgne, :crmr, :lnlq, :craig, :craigmr)
+          solve!(solver, Au, c)
+          niter = niterations(solver)
+          @test Aprod(solver) == niter
+          @test Atprod(solver) == niter
+          @test solution(solver, 1) === solver.x
+          @test nsolution(solver) == (method ∈ (:cgne, :crmr) ? 1 : 2)
+          (nsolution == 2) && (@test solution(solver, 2) == solver.y)
+        end
+
+        if method ∈ (:cgls, :crls, :lslq, :lsqr, :lsmr)
+          solve!(solver, Ao, b)
+          niter = niterations(solver)
+          @test Aprod(solver) == niter
+          @test Atprod(solver) == niter
+          @test solution(solver) === solver.x
+          @test nsolution(solver) == 1
+        end
+
+        if method ∈ (:bilqr, :trilqr)
+          solve!(solver, A, b, b)
+          niter = niterations(solver)
+          @test Aprod(solver) == niter
+          @test Atprod(solver) == niter
+          @test solution(solver, 1) === solver.x
+          @test solution(solver, 2) === solver.y
+          @test nsolution(solver) == 2
+          @test issolved_primal(solver)
+          @test issolved_dual(solver)
+        end
+
+        if method ∈ (:tricg, :trimr, :gpmr)
+          method == :gpmr ? solve!(solver, Ao, Au, b, c) : solve!(solver, Au, c, b)
+          niter = niterations(solver)
+          @test Aprod(solver) == niter
+          method != :gpmr && (@test Atprod(solver) == niter)
+          method == :gpmr && (@test Bprod(solver) == niter)
+          @test solution(solver, 1) === solver.x
+          @test solution(solver, 2) === solver.y
+          @test nsolution(solver) == 2
+        end
+
+        if method ∈ (:usymlq, :usymqr)
+          method == :usymlq ? solve!(solver, Au, c, b) : solve!(solver, Ao, b, c)
+          niter = niterations(solver)
+          @test Aprod(solver) == niter
+          @test Atprod(solver) == niter
+          @test solution(solver) === solver.x
+          @test nsolution(solver) == 1
+        end
+
+        @test niter > 0
+        @test statistics(solver) === solver.stats
+        @test issolved(solver)
+      end
+
+      io = IOBuffer()
+      show(io, solver, show_stats=false)
```

Comment:
```suggestion
      show(io, solver, show_stats=true)
```
just to increase coverage :).

Reply:
You should look at line 147 ;)